interface/src/i18n/se/index.ts

```diff
@@ -174,6 +174,10 @@ const se: BaseTranslation = {
   SYSTEM_FACTORY_TEXT_DIALOG: 'Är du säker att du vill fabriksåterställa enheten?',
   VERSION_CHECK: 'Kontrollera version',
   THE_LATEST: 'Den senaste',
+  OFFICIAL: 'official',
+  DEVELOPMENT: 'development',
+  /**VERSION_IS: 'version is',*/
```

Comment:
Afaik the language fallback of i18n is on file level, not on word level. We need all words defined or it will be empty. Better use the english text and add comment // translation needed! 
In SE there is also RESTART_CONFIRM from earlier missing. The different line nubers of the index files are irritating, maybe use the same formatting for each index.ts to see differences better.

Reply:
I noticed that there is no translation for VERSION_IS in SE. Not knowing if it was by mistake or on purpose, I put the entire line from EN as a comment

Reply:
So why not to do something like this: https://github.com/ivanhofer/typesafe-i18n#i-have-two-similar-locales-only-a-few-translations-are-different-but-i-dont-want-to-duplicate-my-translations ?
Every missed/forgotten word missed would be automatically replaced by English one. We could even remove (or comment out) words that we doesn't want to be translated...